website/src/routes/routes.ts

```diff
@@ -27,13 +28,10 @@ export const routes = {
         `/seq/${getAccessionVersionString(accessionVersion)}`,
     sequencesVersionsPage: (accessionVersion: AccessionVersion | string) =>
         `/seq/${getAccessionVersionString(accessionVersion)}/versions`,
-    sequencesFastaPage: (accessionVersion: AccessionVersion | string, download = false) => {
-        let url = `${routes.sequencesDetailsPage(accessionVersion)}.fa`;
-        if (download) {
-            url += '?download';
-        }
-        return url;
-    },
+    sequencesFastaPage: (accessionVersion: AccessionVersion | string, download = false) =>
```

Comment:
I see this is in existing code but I think we shouldn't put this in the plural (sequences). I would probably change these 4 items to `sequenceEntry[..]`

Reply:
good point! fixed: https://github.com/loculus-project/loculus/pull/3003/commits/4650614bfa3f4665d0e9ecd5b331fc5369221b17